packages/ember-data/lib/system/relationships/belongs-to.js

```diff
@@ -93,6 +93,14 @@ function belongsTo(modelName, options) {
 
   opts = opts || {};
 
+  if (opts.hasOwnProperty('serialize')) {
+    Ember.warn('You provided a serialize option, this belongs in the serializer. See DS.Serializer and it\'s implementations http://emberjs.com/api/data/classes/DS.Serializer.html');
```

Comment:
Can you tell the user for which model they provided it, that would make the warning much less confusing. Something like: In the `modelName` class,  passed a serialize option to the belongsTo relationship `key`. You most likely want to be configuring that in the serializer...